orchagent/port.h

```diff
@@ -85,7 +85,7 @@ class Port
     sai_object_id_t     m_ingress_acl_table_group_id = 0;
     sai_object_id_t     m_egress_acl_table_group_id = 0;
     vlan_members_t      m_vlan_members;
-    sai_port_oper_status_t m_oper_status = SAI_PORT_OPER_STATUS_UNKNOWN;
+    sai_port_oper_status_t m_oper_status;
```

Comment:
This brings back undefined behavior, e.g:
For LAG, VLAN m_oper_status field is never set, so there is a chance that NeighOrch will invalidate neighbor on LAG or VLAN in next hop group in case m_oper_status equals SAI_PORT_OPER_STATUS_DOWN.